cloud/cloud_file_deletion_scheduler.h

```diff
@@ -39,6 +39,7 @@ class CloudFileDeletionScheduler
     std::vector<std::string> files;
     for (auto& [file, handle] : files_to_delete_) {
       files.push_back(file);
+      (void)handle;
```

Comment:
What is this for?

Reply:
Fixing the compile, handle is unused and compiler doesn't like it.